internal/model/exists.go

```diff
@@ -0,0 +1,5 @@
+package model
+
+type Exists struct {
```

Comment:
where is this used?

Reply:
at the moment nowhere, because project has no unique checks, its used for this